cpp/sopt/proximal_expression.h

```diff
@@ -6,9 +6,7 @@
 #include <Eigen/Core>
 #include "sopt/maths.h"
 
-namespace sopt {
-//! Holds some standard proximals
-namespace proximal {
+namespace sopt::proximal {
 
```

Comment:
Again I think we should keep the comment for docs.

Reply:
Yes. Agreed. Will fix and update the PR. Thank you.

Reply:
Thank you for pointing this out. Fixed.